### DISCUSSION TITLE
PantherX2 卡刷 armbian ssh 登录失败

### DISCUSSION THREAD

**Discussion by happyelements01** [ophub/amlogic-s9xxx-armbian#2136] [Rockchip]
卡刷后能找到 ip，ssh root@192.168.123.101 输入密码 1234 无法登录显示 Permission denied

加 -v 参数能发现一个错误为 debug1: read_passphrase: can't open /dev/tty: No such file or directory

有大佬知道是什么原因吗

**Comment by ophub** [comment]
重新刷写固件

**Reply by happyelements01** [reply]
我下载最新的刷试试

**Comment by zeyios** [comment]
我也是这个情况，路由器上显示的是Panther-X2，ssh密码不对，感觉是卡刷没有刷成功，可能还是原来的系统，所以登录不上去。。
但是我线刷试了reset和短接又都没有反应。。

@happyelements01 有搞定吗？

**Reply by happyelements01** [reply]
我线刷成功在路由器现实的是 armbian，不是 Panther-X2，还是以前老系统

你新机线刷失败是不是，你参照这个刷

![image](https://github.com/ophub/amlogic-s9xxx-armbian/assets/11953672/fa4cbfcb-ea67-456f-a465-134da25580f8)
我用上面这样子的工具线刷成功了

https://www.cnblogs.com/chuixifeng/p/17940075

**Reply by zeyios** [reply]
谢谢，买了新的刷机线，回头再试试。  
之前是直接用的手机数据线，typec那头插电脑上，电脑一直都显示没有发现设备。
